lib/parser/rules.bnf

```diff
@@ -0,0 +1,190 @@
+expression ::= condition_operator primary
```

Comment:
These rules are not used in the expression parser and are not relevant imo. Delete the file?